pages/work.tsx

```diff
@@ -0,0 +1,28 @@
+import Head from "next/head";
+import Subscription from "@/components/general/Subscription";
+import WorkTitle from "@/components/Work/WorkTitle";
+import WorkPosts from "@/components/Work/WorkPosts";
+
+export default function Work() {
```

Comment:
Just like the blog page, I feel like this page should be the **index.tsx** in **work** folder instead of another **work.tsx** file